app/presenters/bank_transaction_presenter.rb

```diff
@@ -9,6 +9,8 @@ class BankTransactionPresenter
     running_balance: 'balance/running total',
     account_type: 'account type',
     account_name: 'account name',
```

Comment:
For a refactor, couldn't CELLS just be an array of syms instead and then you have the following methods. Saves the duplication of keys and values.
```
  def self.headers
    CELLS.map(&:to_s)
  end

  def build_transaction_hash
    values = {}
    CELLS.map { |k| values[k.to_sym] = send("transaction_#{k}") }
    values
  end
```